server/routes/post-hire-status.ts

```diff
@@ -26,23 +26,38 @@ router.post(
     const user = userId || localUserId;
     // Validate the request body
     await validate(ParticipantPostHireStatusSchema, body);
-    const { participantId } = body;
+    const { participantIds } = body;
     // Check participant exists
-    const [participant] = await getParticipantByID(participantId);
-    if (!participant) {
+    const notParticipants = [];
```

Comment:
Both of these arrays should ideally have typing, but especially notParticipants.

Reply:
Ideally `participantIds` should also have typing, but at least giving `notParticipants` a type is a big improvement.

Reply:
Updated!

Reply:
Looks good!